lib/dl_core/dl_core/united_storage_client_aio.py

```diff
@@ -130,14 +129,17 @@ def __init__(
         self._retry_intervals = (0.5, 1.0, 1.1, 2.0, 2.2)
         self._retry_codes = {408, 429, 500, 502, 503, 504}
 
-        ssl_context = ssl.create_default_context(cafile=get_root_certificates_path())
-
-        self._session = aiohttp.ClientSession(
+        self._bi_http_client = BIAioHTTPClient(
+            base_url="/".join([self.host, self.prefix]),
+            retrier=PredefinedIntervalsRetrier(
+                retry_intervals=self._retry_intervals,
+                retry_codes=self._retry_codes,
+                retry_methods={"GET", "POST", "PUT", "DELETE"},  # TODO: really retry all of them?..
```

Comment:
retriing non-indempotent methods sounds scary

Reply:
yes, it's kinda strange
https://github.com/datalens-tech/datalens-backend/issues/249

I was afraid to change it without discussion